lantiq-xrx200: fix wifi LED on o2 box 6431

maurerle opened the pull request:
wifi LED did not work using phy0radio, which somehow slipped through in the previous testing

Signed-off-by: Florian Maurer <f.maurer@outlook.de>

blocktrron commented on target/linux/lantiq/xrx200/base-files/etc/board.d/01_leds:
```diff
@@ -30,7 +30,7 @@ arcadyan,arv7519rw22)
 arcadyan,vgv7510kw22-brn)
 	ucidef_set_led_netdev "internet" "internet" "$led_internet" "wan"
-	ucidef_set_led_wlan "wifi" "wifi" "green:wlan" "phy0radio"
+	ucidef_set_led_wlan "wifi" "wifi" "green:wlan" "phy0tpt"
```
The default functionality should be defined in the devices DTS instead of userspace.

maurerle replied:
I don't know how this would look like?
I don't see where the should be set in:
https://github.com/openwrt/openwrt/blob/master/target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vgv7510kw22.dtsi
or
https://github.com/openwrt/openwrt/blob/master/target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vgv7510kw22-nor.dts

Maybe you can give me a hint of which change is needed here? @blocktrron

blocktrron replied:
Look at the ath79 target (OCEDO Koala / OCEDO Racoon for example). The property is set for the LED node.

maurerle replied:
Thanks.
Should work now?

blocktrron replied:
This still requires to delete the existing Userspace configuration creation.

maurerle replied:
Thanks, of course! :)

maurerle committed:
lantiq: xrx200: Fix wifi LED on o2 box 6431

Wifi LED did not work using phy0radio, which somehow slipped through in
the previous testing

Signed-off-by: Florian Maurer <f.maurer@outlook.de>
